scene/3d/sprite_3d.cpp

```diff
@@ -360,6 +383,23 @@ void Sprite3D::_draw() {
 
 	float pixel_size = get_pixel_size();
 
+	if (use_anchor) {
+		Point2 calc_anchor = anchor;
+
+		if (centered) {
+			calc_anchor.x = CLAMP(anchor.x, -0.5, 0.5);
+			calc_anchor.y = CLAMP(anchor.y, -0.5, 0.5);
+		}
+
+		Size2 tex_size;
+		if (region)
+			tex_size = region_rect.size;
+		else
+			tex_size = Size2(texture->get_size()) / Size2(hframes, vframes);
+
+		offset = Point2(tex_size.width * calc_anchor.x, tex_size.height * -calc_anchor.y);
```

Comment:
Yeah, I should have notified the editor about the changes in offset... Whoops :cold_sweat: 

On a slight aside, do 3D sprites work for you? I wasn't able to get them to work, so I couldn't test if the anchor code was working.